[T7][T16-C2]

Halo3fanz opened the pull request:
Readme, User Guide and .project updates

PierceAndy commented on docs/AboutUs.md:
```diff
@@ -1,52 +1,34 @@
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Ong Li-Jen]() <br>
+<img src="images/" width="150"><br>
```
- Please upload a recent photo of yourself.
- Do not put what should have been an issue/task into a commit message (i.e. "... please add your picture to images ...")

PierceAndy commented on docs/UserGuide.md:
```diff
@@ -121,6 +121,9 @@ There is no need to save manually.
        the file that contains the data of your previous Address Book folder.
        
+**Q**: How do I report an issue?
+**A**: Contact e0008233@u.nus.edu for assistance.
```
It would make more sense for the user to open an issue on github.

PierceAndy commented on .project:
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="0.0" encoding="UTF-8"?>
```
http://stackoverflow.com/questions/13743250/meaning-of-xml-version-1-0-encoding-utf-8

PierceAndy commented on docs/AboutUs.md:
```diff
@@ -1,52 +1,34 @@
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Ong Li-Jen]() <br>
```
Please add a link to your own webpage / github user page.

PierceAndy commented:
- Commit messages should be in imperative mood (e.g. "Change" instead of "Changes")
- Commit messages should be descriptive, such that others viewing your code can understand what you're trying to achieve in a commit. They should also be in the imperative form. Try taking a look at your peers' commit messages to get an idea of what makes a good or bad commit message.
- This PR does not reach the requirements for V0.0 or V0.1. The user guide has not been updated meaningfully. The readme does not acknowledge the original source of the code.

Some comments added. Please close the PR after reading comments.

lijen1994 committed:
Merge pull request #1 from CS2103AUG2016-T16-C2/v0.3

V0.3

Halo3fanz committed:
Merge pull request #46 from CS2103AUG2016-T16-C2/AddressToTask

Update MainApp.java

Halo3fanz committed:
Add preliminary code for ClashCommand

Commit:
Remove DeadlineQualifier

lijen1994 committed:
Update Images

update files with incorrect diagrams

lijen1994 committed:
Update DeveloperGuide.md

add fixed images
change formatting and grammar

lijen1994 committed:
Update UserGuide.md

Change appearances 
Add page breaks
Edit content

lijen1994 committed:
Add UG & DG PDF's

Halo3fanz committed:
Add ClashCommand. Add contains functionality to TaskManager

Halo3fanz committed:
Merge pull request #48 from CS2103AUG2016-T16-C2/UGpatch1

Update UserGuide.md

Halo3fanz committed:
Merge pull request #47 from CS2103AUG2016-T16-C2/DGpatch1

Update DeveloperGuide.md

lijen1994 committed:
Update DeveloperGuide.md

Add MSS description

Commit:
Add test for Clear command. Change message that Clear command outputs

Halo3fanz committed:
Fix indentation

Halo3fanz committed:
Fix #43

Fix #43 - Clash

AnniyaBK committed:
Add floating tasks

AnniyaBK committed:
-Edit the code in Edit Command
-Allow deadline to be displayed

AnniyaBK committed:
Edit storage files to adapt to the optional deadline environment

AnniyaBK committed:
Store deadline tasks after implementing floating tasks

AnniyaBK committed:
Solve merge conflict

lijen1994 committed:
Merge pull request #54 from AnniyaB/v0.3

Add floating task

Halo3fanz committed:
Merge pull request #3 from CS2103AUG2016-T16-C2/v0.3

Anniya's Floating Task

Commit:
Fix conflict with ClashCommand and Floating Tasks

lijen1994 committed:
Merge pull request #55 from Halo3fanz/master

Fix #27

e0008233 committed:
change of travis status

e0008233 committed:
change of travis status

e0008233 committed:
change of travis status

e0008233 committed:
Merge pull request #56 from e0008233/v0.3

change of travis status

e0008233 committed:
Change badges to Travis and Coverall

e0008233 committed:
change of coverall status

Halo3fanz committed:
Add preliminary Timed Task functionality.

e0008233 committed:
change of travis status

Commit:
Update UG, XMLAdaptedPerson, and xml files to match new AddCommand

Commit:
Fix Parser to allow blank Startline

lijen1994 committed:
Merge pull request #65 from Halo3fanz/master

Fix #44 Add Timed Task functionality.

Halo3fanz committed:
Solve all issues with tests

Halo3fanz committed:
Add edit TimedTask functionality

lijen1994 committed:
Merge pull request #67 from Halo3fanz/master

Fix #45 Edit Timed Task Functionality and Error Fixes

Halo3fanz committed:
Remove some unnecesssary print blocks

lijen1994 committed:
Merge pull request #68 from Halo3fanz/master

Remove some unnecesssary print blocks

AnniyaBK committed:
Enhance floating task implementation

AnniyaBK committed:
Adapt ReadOnlyTask to optional priority

lijen1994 committed:
Merge pull request #70 from AnniyaB/v0.4

Enhance floating task implementation

lijen1994 committed:
Update README.md

Update Coverall

Commit:
Add of completeCommand and listallCommand

e0008233 committed:
Merge pull request #72 from e0008233/v0.3

Add of completeCommand and listallCommand

e0008233 committed:
Merge pull request #73 from e0008233/V0.4_testing

add of completeCommand and listAllCommand

Commit:
Refactor CompleteCommand and editCommand

e0008233 committed:
Merge pull request #74 from e0008233/v0.4

Refactor of CompleteCommand and editCommand

Commit:
Add time functionality to Deadline

Halo3fanz committed:
Fix Parser

Halo3fanz committed:
Fix Parser

Halo3fanz committed:
Fix Parser

Halo3fanz committed:
Fix ModelManager

Halo3fanz committed:
Fix ModelManager

Halo3fanz committed:
Fix ModelManager.java

lijen1994 committed:
Merge pull request #75 from Halo3fanz/master

Add time functionality to Deadline

Halo3fanz committed:
Merge pull request #4 from CS2103AUG2016-T16-C2/v0.4

Update fork

Commit:
Fix Complete Command

Commit:
Reimplement optional priority

lijen1994 committed:
Merge pull request #77 from Halo3fanz/master

Fix Complete Command

Commit:
Add preliminary RepeatCommand

Commit:
add of TagQualifier in ModelManager.java

Commit:
add of updateFilteredPersonGroup

Commit:
add of GroupCommand class

Commit:
change of Success message in GroupCommand

Commit:
add of GroupCommand in Model interface

Commit:
add of GroupCommand in case of Parser.java

Commit:
add of prepareGroup in Parser.java

Commit:
add of description in GroupCommand

Commit:
change of task size in completeCommand in ModelManager.java

lijen1994 committed:
Update UserGuide.md

Reformat UserGuide to comply with CS2101 UG
Add complete command
Add listall command
Update list features
Update command formatting for add

Commit:
delete of unused imports in ModelManager

Commit:
refractor in the ModelManager

AnniyaBK committed:
Merge pull request #82 from CS2103AUG2016-T16-C2/userGuide-v0.4.01

Update UserGuide.md

lijen1994 committed:
Update UserGuide.md

Remove N.A from commands without the need for examples

lijen1994 committed:
Update GroupCommand.java

Add Clarification on the purpose of group

lijen1994 committed:
Merge pull request #83 from e0008233/v0.4

add of GroupCommand

lijen1994 committed:
Merge pull request #2 from CS2103AUG2016-T16-C2/v0.4

Current State

Commit:
change of formatting of EditCommand

Commit:
change of indentation in the Parser.java

Commit:
change of floating format for EditCommand

Commit:
Change of CompleteCommand and not allow the same task to be completed
twice.

Commit:
Fix for FindCommand (case insensitive, multiple keywords, partial
search)

Halo3fanz committed:
Add Repeat Command. Add preliminary Update Command.

e0008233 committed:
Merge pull request #90 from e0008233/v0.4

1. Change of EditCommand to allow floating format 2. Change of ComplementCommand to not allow the same task to be completed twice 3. Fix of FindCommand to allow more flexible search

e0008233 committed:
Update EditCommand.java

Commit:
Add feature where adding or editing at task makes the list jump to the latest task

Commit:
Add preliminary UpdateCommand

e0008233 committed:
Merge pull request #102 from e0008233/v0.4

Update EditCommand.java

lijen1994 committed:
Merge pull request #3 from CS2103AUG2016-T16-C2/v0.5

V0.5

lijen1994 committed:
Undo and Revert update

AnniyaBK committed:
Merge pull request #103 from lijen1994/v0.5

Undo and Revert with corrected files

Halo3fanz committed:
Add Overdue, Repeat and Update functionality. Fix Complete Command to be unable to be repeated in interim

lijen1994 committed:
Change Help URL to most recent edit
Made dialogStage have less white space

Halo3fanz committed:
Fix AddCommand

lijen1994 committed:
Merge pull request #105 from lijen1994/v0.5

Update HelpWindow.java

Halo3fanz committed:
Update AddCommand.java

Halo3fanz committed:
Update AddCommand.java

Halo3fanz committed:
Update AddCommand.java

Halo3fanz committed:
Update AddCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update EditCommand.java

Halo3fanz committed:
Update Parser.java

Halo3fanz committed:
Update Parser.java

Halo3fanz committed:
Add new Parser.java

Halo3fanz committed:
Add new Parser.java

Halo3fanz committed:
Fix EditCommand.java

lijen1994 committed:
Merge pull request #104 from Halo3fanz/master

Fix #33, fix #78

Halo3fanz committed:
Merge pull request #5 from CS2103AUG2016-T16-C2/v0.5

V0.5 Update

Halo3fanz committed:
Fix issue where "monthly" tag is not removed. Add ablility for repeat command to switch time intervals and be turned off with a specific keyword. Add meaningful comments to RepeatCommand.java and Parser.java. Add functionality where list scrolls to repeated task on sucess.

Halo3fanz committed:
Fix MESSAGE_USAGE to reflect new RepeatCommand.java functionality

e0008233 committed:
Merge pull request #108 from Halo3fanz/master

Fix #106

Halo3fanz committed:
Fix #107 - Allow Completed Tasks to be repeated

lijen1994 committed:
Merge pull request #109 from Halo3fanz/master

Fix #107 - Allow Completed Tasks to be repeated

lijen1994 committed:
Merge pull request #5 from CS2103AUG2016-T16-C2/v0.5

New Pull

e0008233 committed:
change of address book

e0008233 committed:
change of language

lijen1994 committed:
Update UserGuide.md

Added Undo and Revert commands to the user guide.
Added update command to the UG

lijen1994 committed:
Update user guide

Halo3fanz committed:
Add Colour Codes for upcoming deadlines. Fix Startline where time is improperly stored due to faulty regex. Add DateManager.java that handles Calendar dates.

lijen1994 committed:
Merge pull request #111 from Halo3fanz/master

Add colour codes for upcoming deadlines.

Halo3fanz committed:
Add preliminary Update.java

Commit:
Change persons to tasks

Commit:
Add ScrollCommand.java. Allow user to scroll through the list

e0008233 committed:
Merge pull request #112 from Halo3fanz/master

Add ScrollCommand.java

Commit:
Add ability for task manager to update itself every minute

Commit:
Add Confirmation Dialog for Clear Command

Commit:
Remove redundant Update.java

Commit:
Remove start, end and priority labels if null

lijen1994 committed:
Merge pull request #115 from Halo3fanz/master

Add Update functionality and dialog for ClearComand

Commit:
Remove icon from clear dialog

Commit:
add of configure to Command.java, add of SaveCommand

Commit:
change of SaveCommand

Commit:
add of invalid file path message

lijen1994 committed:
Update AddCommand.java

lijen1994 committed:
Update DeleteCommand.java

lijen1994 committed:
Update CompleteCommand.java

lijen1994 committed:
Update FindCommand.java

lijen1994 committed:
Update Parser.java

lijen1994 committed:
Update AddCommand.java

lijen1994 committed:
Update CompleteCommand.java

Commit:
add of LoadCommand

Commit:
add of LoadCommand

Commit:
update of SaveCommand

lijen1994 committed:
Merge pull request #118 from e0008233/v0.5

add SaveCommannd and LoadCommand

e0008233 committed:
Merge pull request #117 from lijen1994/shortcuts

Shortcuts and UG updates

e0008233 committed:
Merge pull request #1 from CS2103AUG2016-T16-C2/v0.5

update of all

lijen1994 committed:
Update CompleteCommand.java

Commit:
add exception to invalid file path

e0008233 committed:
Merge pull request #120 from e0008233/v0.5

add invalid file path exception

AnniyaBK committed:
Fix error in LogicManager test

AnniyaBK committed:
Add in clash and edit commands

AnniyaBK committed:
Delete main

AnniyaBK committed:
Did edit and clash commands

AnniyaBK committed:
Update collate

AnniyaBK committed:
Delete A0139097U

AnniyaBK committed:
Update collate

AnniyaBK committed:
Update collate

AnniyaBK committed:
Update collate

AnniyaBK committed:
Update collate

AnniyaBK committed:
Update collate

AnniyaBK committed:
Edit UG and DG

AnniyaBK committed:
Delete docs

AnniyaBK committed:
Edit UG and DG

AnniyaBK committed:
Edit UG and DG

AnniyaBK committed:
Collate all files

Halo3fanz committed:
Merge pull request #128 from AnniyaB/v0.5

Collate all files

Commit:
Add logger output to repeating update command

Commit:
Fix UpdateCommand.java

Commit:
Fix #110

Commit:
Fix floating tasks clashing with each other

AnniyaBK committed:
Merge pull request #129 from Halo3fanz/master

Bug Fixes

Halo3fanz committed:
Merge pull request #6 from CS2103AUG2016-T16-C2/v0.5

Update Remote Repo

Halo3fanz committed:
Fix UpdateCommand where repeating tasks were set to overdue by mistake

Halo3fanz committed:
Fix an instance where UpdateCommand.java does not return the minute value in the correct way.

Halo3fanz committed:
Fix issue where Repeating class was not stored by Task correctly

Halo3fanz committed:
Remove some println statements from UpdateCommand.java

Halo3fanz committed:
Fix #130

lijen1994 committed:
TestPictureFile

Command list help direct

lijen1994 committed:
Delete 2016-10-23 (3).png

Halo3fanz committed:
Fix #131

e0008233 committed:
Merge pull request #134 from Halo3fanz/master

Fix #130, fix #131

lijen1994 committed:
Update README.md

lijen1994 committed:
Update DeveloperGuide.md

Commit:
made groupCommand case insensitive

lijen1994 committed:
Update README.md

Fix content
Add more relevant details

Commit:
change GroupCommand to ListtagCommand

e0008233 committed:
Merge pull request #135 from e0008233/v0.5

Make listtag case insensitively

Halo3fanz committed:
Fix ScrollCommand's MESSAGE_USAGE

AnniyaBK committed:
Update AboutUs.md

Halo3fanz committed:
Fix name input for EditCommand.java

AnniyaBK committed:
Update User Stories

lijen1994 committed:
Merge pull request #136 from Halo3fanz/master

Fix name input with EditCommand.java and ScrollCommand.java's MESSAGE_USAGE

AnniyaBK committed:
Add specific references to the graphics in the commentary

lijen1994 committed:
Add new icons via upload

New icons for TPTM to replace addressbook icons

AnniyaBK committed:
Update UserGuide.md

AnniyaBK committed:
Update UserGuide.md

AnniyaBK committed:
Delete LogicClassDiagram.png

AnniyaBK committed:
Add Logic Class Diagram

Commit:
Refractor of model.java

Commit:
refractor

Commit:
Refractor

Commit:
Refractor

e0008233 committed:
Merge pull request #137 from e0008233/v0.5

Refractoring

lijen1994 committed:
Update UserGuide.md

Final Edit

AnniyaBK committed:
Delete UI image

AnniyaBK committed:
Add new Ui image

AnniyaBK committed:
Update UserGuide.md

Commit:
refractor

AnniyaBK committed:
Update README.md

Commit:
refractor

lijen1994 committed:
Update DeveloperGuide.md

Change Appendix E to a table
Edit some minor points

AnniyaBK committed:
Merge pull request #138 from e0008233/v0.5

refractor

Commit:
refractor

e0008233 committed:
Merge pull request #139 from e0008233/v0.5

xml file

Halo3fanz committed:
Add data file folder

Halo3fanz committed:
add ManualTesting file folder

e0008233 committed:
data file

AnniyaBK committed:
Update Collate

e0008233 committed:
Add files via upload

e0008233 committed:
Merge pull request #140 from AnniyaB/v0.5

Update Collate

lijen1994 committed:
Add files via upload

lijen1994 committed:
Delete Shortcut Icon.png

Halo3fanz committed:
Fix address_book icon

AnniyaBK committed:
Update collate

lijen1994 committed:
Merge pull request #141 from AnniyaB/v0.5

Update collate

lijen1994 committed:
Update README.md

AnniyaBK committed:
Update collate

lijen1994 committed:
Merge pull request #142 from AnniyaB/v0.5

Update collate

AnniyaBK committed:
Delete A0139516B

AnniyaBK committed:
Delete A0141812R

lijen1994 committed:
Update DeveloperGuide.md

AnniyaBK committed:
Delete br.md

AnniyaBK committed:
Delete A0139097U

AnniyaBK committed:
Delete A0139516B

AnniyaBK committed:
Delete A0141812R

AnniyaBK committed:
Delete A0144202Y

AnniyaBK committed:
Delete A0139097U

AnniyaBK committed:
Delete A0141812R

Halo3fanz committed:
Fix ManualTesting folder location

Changed from /data to /src/test/data

e0008233 committed:
update of ManualTesting

e0008233 committed:
update pf test script

e0008233 committed:
Update TestScript.md

lijen1994 committed:
Update TestScript.md

lijen1994 committed:
Update TestScript.md
